mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.tsx

```diff
@@ -36,7 +36,7 @@ interface DropdownMenuItemProps {
   /**
    * Items`s value
    */
-  value: string;
+  value: ReactNode;
```

Comment:
Looks like React node is to wide interface, could we create few specific types for this puprpose to keep coonsistency for dropdown menu items. Maybe it will be <DropdownItem .../> or just config for it like {valus: string, icon: IconName}

Reply:
what do you think

Reply:
I rolled back to use "string"